jepsen/src/jepsen/nemesis/combined.clj

```diff
@@ -359,6 +359,60 @@
                            :fs    #{:strobe-clock}
                            :color "#A0E9E3"}}}))
 
+(defn file-corruption-package
+  "A nemesis and generator package that corrupts files.
+   
+   Opts:
+   ```clj
+   {:file-corruption
+    {:targets     [...] ; A collection of node specs, e.g. [:one, [\"n1\", \"n2\"], :all]
+     :corruptions [     ; A collection of file corruptions, e.g.:
+      {:bitflip \"/path/to/file\"
+       :probability 1e-3},
+      {:bitflip \"path/to/dir\"
+       :probability [1e-3 1e-4 1e-5]},
+      {:truncate \"path/to/file/or/dir\"
+       :drop (range 1 1025)}]}}
```

Comment:
This `(range)` concerns me a bit, because it's easy to imagine someone actually materializing and iterating over, say, (range 1 1e10), which... could be disastrous, especially if it leaks into a pretty-printer somewhere. You also might not want a uniform distribution over the choices--you might want to lose just one byte sometimes, and hundreds of megs other times, and a uniform distribution would almost never select small truncations. Maybe something like

```clj
{:distribution :geometric
 :mean 500}
```

or 

```clj
{:distribution :uniform
 :min 50
 :max 5000}
```

Reply:
Yes, it's  problematic Clojure and only allows for a uniform distribution.

Will commit and summarize in main conversation thread.